proposals/gc/Post-MVP.md

```diff
@@ -403,6 +403,16 @@ However, there are a number of challenges:
 
 In general, the semantics and implementation of type parameters should be analogous to that of type imports. Ideally, in the presence of the [module linking proposal](https://github.com/WebAssembly/module-linking), it should even be possible to explain definitions with type parameters as shorthands for nested modules (well, at least for 2nd-class cases).
 
+### Fundamental Limitations
+
+Unfortunately, languages with generics exhibit one or both of the following features:
```

Comment:
As written, this is a false statement. Ocaml would be an obvious counter example, which requires uniform recursion for both structural and nominal types and has no bounded quantification.

Reply:
OCaml appears to only restrict recursive polymorphic functions to be uniformly recursive. The following expansive-recursive type seems to compile in OCaml: `type 'a t = EMPTY | NODE of 'a * (('a t) t)`.